code/e2e-tests/framework-nextjs.spec.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable jest/no-disabled-tests */
+import type { Locator } from '@playwright/test';
+import { test, expect } from '@playwright/test';
+import process from 'process';
+import { SbPage } from './util';
+
+const storybookUrl = process.env.STORYBOOK_URL || 'http://localhost:6006';
+const templateName = process.env.STORYBOOK_TEMPLATE_NAME;
+
+test.describe('Next.js', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(storybookUrl);
+    await new SbPage(page).waitUntilLoaded();
+  });
+
+  test.describe('next/navigation', () => {
+    test.skip(
```

Comment:
I wonder what this will do to the CI status report? cc @kasperpeulen

Reply:
I guess everything will fail. But @kasperpeulen is working on adding metadata to tests, so I hope there's a way to configure this, either by adding the skip descrition or something, which allows us to get the results right